evap/static/scss/components/_callouts.scss

```diff
@@ -94,10 +94,19 @@
 }
 
 .infobox.callout {
-  max-width: 100%;
+    min-width: 100%;
 
-  // enlarge infobox to max size
-  .callout-content {
-    width: 1300px;
-  }
```

Comment:
Afaik, in CSS, pixels [unintuitively are an absolute unit](https://drafts.csswg.org/css-values/#absolute-lengths), so I think 1300px wasn't really a problem here. But I'm fine with both the old and the new code.